sdk/openai/azure-ai-openai/src/test/java/com/azure/ai/openai/NonAzureOpenAIAsyncClientTest.java

```diff
@@ -117,7 +129,10 @@ public void testGetCompletionsBadSecretKey(HttpClient httpClient, OpenAIServiceV
     @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
     @MethodSource("com.azure.ai.openai.TestUtils#getTestParameters")
     public void testGetCompletionsExpiredSecretKey(HttpClient httpClient, OpenAIServiceVersion serviceVersion) {
-        client = getNonAzureOpenAIAsyncClient(httpClient);
+        client = getNonAzureOpenAIAsyncClient(
+            httpClient,
+            new NonAzureOpenAIKeyCredential("sk-123456789012345678901234567890123456789012345678"));
```

Comment:
Is possible we keep using "not_token_looking_string" instead of this fake credential (it looks like it is a possible credential)?
I initially thought it will fail the CredScan check but it doesn't. 
(FWIW, if failed with CredScan check we will need to suppress the credential value by following steps:
```
How to suppress the credential value:

Create a client based suppression file, "FakeCredentialConstants". Add the file reference to the global suppression file, "eng/CredScanSuppression.json". 

Note: Placeholders should be very specific, it could possible over suppress that could result in accidentally suppressing true secret.

	• Create a java file, "FakeCredentialConstants.java: in the client's  /implementation folder.
		a. Name the value of credential with suffix,  "placeholder", e.x., "fakeUsernamePlaceholder".

		private static final String COMMUNICATION_MSAL_USERNAME = Configuration.getGlobalConfiguration()
		            .get("COMMUNICATION_MSAL_USERNAME", "fakeUsernamePlaceholder");
		    private static final String COMMUNICATION_MSAL_PASSWORD = Configuration.getGlobalConfiguration()
		            .get("COMMUNICATION_MSAL_PASSWORD", "fakePasswordPlaceholder");
		
	• Add this file reference to "suppressions" array in the eng/CredScanSuppression.json. For example,
	       {
	            "file": [
	                "sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/implementation/FakeCredentialConstants.java"
	            ],
	            "_justification": "File contains fake key used by implementation code."
                },
```
)

Reply:
In this case, we can keep the changes here and will suppress the CredScan if failed.